runtime/src/main/java/org/corfudb/runtime/BootstrapUtil.java

```diff
@@ -138,4 +139,41 @@ public static void bootstrap(@NonNull Layout layout,
         }
         log.info("Successfully Bootstrapped layout:{} .", layout);
     }
+
+    /**
+     * Bootstrap the given layout for all the nodes in the router map with the configured
+     * routers. This implementation ignores the AlreadyBootstrappedException.
+     * @param routerMap A configured router map
+     * @param layout A layout to bootstrap
+     * @param retries Num retries
+     * @param retryDuration Duration between retries
+     */
+    public static void bootstrapWithRouterMap(Map<String, NettyClientRouter> routerMap,
+                                              Layout layout, int retries,
+                                              @NonNull Duration retryDuration) {
+        for (String server : routerMap.keySet()) {
+            int retry = retries;
+            try (NettyClientRouter router = routerMap.get(server)) {
+                while (retry-- > 0) {
+                    try {
+                        log.info("Attempting to bootstrap node: {} with layout:{}", server, layout);
+                        bootstrapLayoutServer(router, layout);
+                        bootstrapManagementServer(router, layout);
+                        break;
+                    } catch (AlreadyBootstrappedException abe) {
+                        log.warn("Node already bootstrapped. Skipping.");
```

Comment:
Can we please add the node and the layout in this line where we are skipping with AlreadyBootstrappedException?